kubernetes-dev-environment.yaml

```diff
@@ -14,8 +14,8 @@ spec:
     spec:
       restartPolicy: Never
       containers:
-        - name: dev-environment
-          image: 896129387501.dkr.ecr.us-west-2.amazonaws.com/allennlp:dev-environment
+        - name: allennlp-dev-env
+          image: 896129387501.dkr.ecr.us-west-2.amazonaws.com/allennlp/allennlp-gpu:latest
```

Comment:
@joelgrus let's discuss in person separate from this PR.  We have this problem with or without this PR being merged.

https://app.asana.com/0/358382652518673/404071492289873